discovery-provider/src/queries/get_feed_es.py

```diff
@@ -291,17 +307,15 @@ def fetch_followed_saves_and_reposts(current_user_id, item_keys):
     ]
 
     founds = esclient.msearch(searches=mdsl)
-    collapsed_reposts = founds["responses"][0]["hits"]["hits"]
-    collapsed_saves = founds["responses"][1]["hits"]["hits"]
```

Comment:
Dang this undoes changes from here: https://github.com/AudiusProject/audius-protocol/pull/3797

Not sure why this was a merge conflict as your version looks unchanged from parent commit... but we should keep the `collapsed_reposts` version.

Reply:
interesting. this may be why the the feed es tests were failing. thanks

Reply:
fixed! and test passes :)